app/wallet/loader.js

```diff
@@ -24,44 +25,44 @@ export const getWalletExists = (loader) =>
     loader.walletExists(new WalletExistsRequest(), (error, response) =>
       error ? reject(error) : resolve(response)));
 
-export const createWallet = (loader, pubPass, privPass, seed) =>
+export const createWallet = log((loader, pubPass, privPass, seed) =>
   new Promise((resolve, reject) => {
     const request = new CreateWalletRequest();
     request.setPrivatePassphrase(new Uint8Array(Buffer.from(privPass)));
     request.setSeed(seed);
     loader.createWallet(request, error => error ? reject(error) : resolve());
-  });
+  }), "Create Wallet", logOptionNoArgs());
 
-export const openWallet = (loader, pubPass) =>
+export const openWallet = log((loader, pubPass) =>
   new Promise((resolve, reject) => {
     const request = new OpenWalletRequest();
     request.setPublicPassphrase(new Uint8Array(Buffer.from(pubPass)));
     loader.openWallet(request, error => error ? reject(error) : resolve());
-  });
+  }), "Open Wallet");
```

Comment:
can you log this with no args as well?  it's logging the attempted public pass being attempted.